app/user_prompt_support/uploaded_file_support.py

```diff
@@ -30,8 +30,7 @@ class UploadFile(Protocol):
     filename: Optional[str]
 
     @property
-    def content_type(self) -> Optional[str]:
-        ...
+    def content_type(self) -> Optional[str]: ...
```

Comment:
```suggestion
    def content_type(self) -> Optional[str]:
        ...
```

Reply:
@rquidute , I was getting additional CI failures due to black, so I locally ran the reformat... I think the repo is in a bad state. What should we do?